### DISCUSSION TITLE
libquery_engine-rhel-openssl not found when running npm install

### DISCUSSION THREAD

**Discussion by GZLiew** [prisma/prisma#22936] [Q&A]
### Question

Hi when I run npm install /node_modules/.prisma/client/libquery_engine-rhel-openssl-1.0.x.so.node does not show in my node modules
im trying to deploy to lambda

### Expected behavior (optional)

/node_modules/.prisma/client/libquery_engine-rhel-openssl-1.0.x.so.node supposed to be found

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```
```
generator client {
  provider     = "prisma-kysely"
  output       = "../src/db"
  fileName     = "types.ts"
  enumFileName = "enums.ts"

  // Specify target required to run in AWS Lambda
  binaryTargets = ["native", "rhel-openssl-1.0.x"]
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model User {
  id                      String                   @id @default(cuid())
  featureSetId            String                   @default("00000000-0000-0000-0000-000000000000")
  emails                  Email[]
  emailSingleUseAuthCodes EmailSingleUseAuthCode[]

  @@map("users")
}

model Email {
  id        String   @id @default(cuid())
  userId    String?
  email     String   @unique
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
  user      User?    @relation(fields: [userId], references: [id])

  @@map("emails")
}

model EmailSingleUseAuthCode {
  id        String    @id @default(cuid())
  userId    String
  code      String
  expiredAt DateTime?
  createdAt DateTime  @default(now())
  user      User      @relation(fields: [userId], references: [id])

  @@map("email_single_use_auth_code")
}

```

- OS: MacOS
- Database: PostgreSQL
- Node.js version: v20.10.0
- Run `prisma -v` to see your Prisma version and paste it
prisma                  : 5.9.1
@prisma/client          : 5.9.1
Computed binaryTarget   : darwin-arm64
Operating System        : darwin
Architecture            : arm64
Node.js                 : v20.10.0
Query Engine (Node-API) : libquery-engine 23fdc5965b1e05fc54e5f26ed3de66776b93de64 (at node_modules/@prisma/engines/libquery_engine-darwin-arm64.dylib.node)
Schema Engine           : schema-engine-cli 23fdc5965b1e05fc54e5f26ed3de66776b93de64 (at node_modules/@prisma/engines/schema-engine-darwin-arm64)
Schema Wasm             : @prisma/prisma-schema-wasm 5.9.0-32.23fdc5965b1e05fc54e5f26ed3de66776b93de64
Default Engines Hash    : 23fdc5965b1e05fc54e5f26ed3de66776b93de64
Studio                  : 0.497.0

**Comment by ludralph** [comment]
Hi @GZLiew 👋 

Thank you for raising this question.

Please try the following steps to see if it helps to resolve your issue.

1. Delete the node_modules and package-lock.json or yarn.lock files in your project.
2. Run npm install or yarn install to reinstall your dependencies.
3. Run `npx prisma generate` to generate the Prisma Client. This should download the correct query engine binary for AWS Lambda based on what you have specified in your `binaryTargets`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
